.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
For future reference, don't check IDE preferences into Git. I'd actually recommend adding this to the `.gitignore` file, untracking this file, then committing.